src/components/sbb-accordion/readme.md

```diff
@@ -1,21 +1,59 @@
-to be documented...
+The `sbb-accordion` is a component which acts as a container for one or more `sbb-expansion-panel`; 
+the content is projected inside the unnamed slot.
+
+### Title level
+
+The component has a `titleLevel` property, which is proxied to the `sbb-expansion-panel-header`, and can be used
+to wrap the header of each `sbb-expansion-panel` in a heading tag; if the property is unset, a `div` is used.
+In the following example, all the `sbb-expansion-panel-header` would be wrapped in a `h3` heading tag.
+
+```html
+<sbb-accordion title-level='3'>
+  <sbb-expansion-panel>
+    <sbb-expansion-panel-header>Header 1</sbb-expansion-panel-header>
+    <sbb-expansion-panel-content>Content 1</sbb-expansion-panel-content>
+  </sbb-expansion-panel>
+  <sbb-expansion-panel>
+    <sbb-expansion-panel-header>Header 2</sbb-expansion-panel-header>
+    <sbb-expansion-panel-content>Content 2</sbb-expansion-panel-content>
+  </sbb-expansion-panel>
```

Comment:
The following suggestions are very nitpicky; Feel free to reject this, if you think it does not fit.
```suggestion
  ...
```

Reply:
Done in [23c4d25](https://github.com/lyne-design-system/lyne-components/pull/1860/commits/23c4d2557ee5eae7ed464f41ad211275b467056a)